lms/templates/problem_notifications.html

```diff
@@ -0,0 +1,17 @@
+<%page expression_filter="h" args="notification_type, notification_icon, submission_response_message, points_earned, points_possible"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+<%
+    notification_type = "partially-correct"
+    notification_icon = "asterisk"
+    submission_response_message = "Partially correct"
+    points_earned = "3"
+    points_possible = "4"
+%>
+
+<div class="submission_response ${notification_type}">
+    <span class="icon fa fa-${notification_icon}" aria-hidden="true"></span><span>${submission_response_message} (${points_earned}/${points_possible} points earned)</span>
+    <button class="btn btn-default btn-small review-btn sr">Review</button>
+</div>
```

Comment:
Also support an optional second action button, to be used for "Next Hint" in the hint usage.